examples/enclosure_group.pp

```diff
@@ -56,6 +56,7 @@
     }
```

Comment:
As a good practice, update the copy right year in examples and lib files.

Reply:
@VenkateshRavula Done!